eth2util/rlp/rlp_test.go

```diff
@@ -58,6 +121,14 @@ func TestBytesList(t *testing.T) {
 	}
 }
 
+func merge(a, b []byte) []byte {
+	resp := make([]byte, len(a)+len(b))
```

Comment:
nit: this can be as simple as `append(a, b...)`

Reply:
no it can't, since that modifies a's underlying array. I specifically wanted to avoid that.